android/src/main/java/com/reactnativekeyboardcontroller/views/EdgeToEdgeReactViewGroup.kt

```diff
@@ -148,7 +163,7 @@ class EdgeToEdgeReactViewGroup(private val reactContext: ThemedReactContext) : R
     Handler(Looper.getMainLooper()).post { view.removeSelf() }
   }
 
-  private fun getContentView(): FitWindowsLinearLayout? {
+  private fun getContentView(): LinearLayout? {
```

Comment:
Because `FitWindowsLinearLayout` can only be accessed from within the same library group prefix (referenced `groupId=androidx.appcompat` with prefix `androidx` from `groupId=...`) More... (⌘F1)